### ISSUE TITLE
Encode symbol cl:null as null

### ISSUE THREAD

**Issue by kilianmh** [phmarek/yason#83]
For some more flexibility, it would be neat to have another method `encode` whith the eql-specifier `cl:null`. This is also used in some other json parsers.
```lisp
(defmethod encode ((object (eql 'cl:null)) &optional (stream *json-output*))
  (write-string "null" stream)
  object)
```
If you like the idea, I will submit the PR.

**Comment by phmarek** [comment]
Yeah, well, I'm not totally against that idea -- but it might interfere with other users of the library in the same image!

Sadly there's no `METHODLET` for local methods - the nearest I've come to that is via an additional context parameter that could be hidden by having an `encode` function that just does `(funcall #'encode-via-context *context* object stream)` ; then, by locally binding the `*context*` variable, arbitrary (local!) behaviour can be implemented.

This would be an incompatible change, though (because `encode` would be a plain function and `encode-via-context` the generic function) -- and I'm unsure about the runtime performance impact.

Perhaps a `yason2` package could do that and just re-use most stuff?
